freshmaker/lightblue.py

```diff
@@ -906,6 +906,11 @@ def find_images_with_included_rpms(
                             "rvalue": tag
                         } for tag in auto_rebuild_tags]
                     },
+                    {
+                        "field": "parsed_data.labels.*.vendor",
+                        "op": "=",
+                        "rvalue": "Red Hat, Inc."
```

Comment:
Do you think this should be configurable? This would allow Freshmaker to be used for other vendors in the future (e.g. partners).